### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by Exploiter-org** [3kh0/ext-remover#1356] [General JOIN TITANIUM NETWORK TO SHARE PLEASE]
Is it possible to unenroll or activate dev mode if I have access to flags and CROSH

**Comment by akabutnicer** [comment]
not at the moment

**Comment by akabutnicer** [comment]
i'm aware you edited your message, but my point remains, assuming you are still on v130

**Comment by Exploiter-org** [comment]
Well does anyone else know

**Reply by akabutnicer** [reply]
i'm aware you deleted your previous message, and made this in replacement. yes there are lower versions that carry exploits, which I will only name a few of:
v105
v111
v114

**Reply by Exploiter-org** [reply]
I'm just looking for CROSH or flags commands that help me get in dev mode or unenroll

**Reply by akabutnicer** [reply]
yeah on what version
